src/app/components/alert/alert.component.html

```diff
@@ -0,0 +1,24 @@
+<div *ngIf="visible" class="adv-alert" [id]=id>
```

Comment:
Is there any reason in particular for why you're using the structural `if` directive over the new control flow syntax here?

Reply:
For a single if-statement, using the traditional syntax avoids one level of indentation. However, for if-else-statements using the new control flow syntax is much more readable.